docs/calculations.mdx

```diff
@@ -313,32 +313,32 @@ creative_data_transfer_bytes =
       ad_format.video_player.size_bytes +
       ad_format.other_asset_bytes
 
-creative_data_transfer_usage_emissions_gco2pm =
+creative_data_transfer_usage_emissions_gco2 =
       creative_data_transfer_bytes / 1024 x
       usage_kwh_per_gb / 1000000 x
       lookup_carbon_intensity_gco2e_per_kwh(country, region, utc_datetime)
 
-creative_data_transfer_embodied_emissions_gco2pm =
+creative_data_transfer_embodied_emissions_gco2 =
```

Comment:
If `creative_data_transfer_bytes` is already multiplied out by impressions/loads then this wouldn't be `pm`

Reply:
right